static/src/javascripts/projects/common/modules/ui/first-pv-consent-banner.spec.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import { banner, _ as test } from './first-pv-consent-banner';
+
+const getAdConsentState: any = require('common/modules/commercial/ad-prefs.lib')
+    .getAdConsentState;
+const setAdConsentState: any = require('common/modules/commercial/ad-prefs.lib')
+    .setAdConsentState;
+const Message: any = require('common/modules/ui/message').Message;
+
+afterEach(() => {
+    jest.clearAllMocks();
+    setAdConsentState(0, null);
+    setAdConsentState(1, null);
+});
+beforeEach(() => {
+    Message.mockReset();
+    Message.prototype.show = jest.fn(() => true);
+    Message.prototype.hide = jest.fn(() => true);
+    Message.prototype.show.mockRestore();
```

Comment:
Previously `Message.prototype.show.mockRestore();` was in a duplicate `afterEach` did you mean to move it into the `beforeEach` when you removed the duplicate `afterEach` block?

Reply:
🙀 Yes! Nice catch. To be honest, this is all a copy paste job from other tests so let me put some order in this area, I think all of this belongs in `beforeEach()`